### DISCUSSION TITLE
About model name

### DISCUSSION THREAD

**Discussion by jet10000** [prisma/prisma#4771] [Q&A] [answered]
```prisma
model blog_post{
   id String @id @default(cuid())
   title String
   content String
}
```

I expect to be able to prefix the table name

This is fine, but is it a best practice?

**Comment by ryands17** [comment]
Hey @jet10000 👋 

I wasn't able to understand the question. Could you explain a bit more about what you're trying to achieve?

If you want to keep the name like this in your database then you can use `@@map` to achieve this as shown [here](https://www.prisma.io/docs/concepts/components/prisma-client/configuring-the-prisma-client-api#using-map-and-map-to-rename-fields-and-models-in-the-prisma-client-api) in the docs.

So your model would look like this:

```prisma
model BlogPost {
   id String @id @default(cuid())
   title String
   content String

  @@map('blog_post')
}
```

And now the table name in the database will be the one specified in `@@map`.

**Reply by jet10000** [reply]
I want to know whether the name of the model must be capitalized

In addition, does the name blog_post conform to the specification in prisma?

**Reply by ryands17** [reply, marked as answer]
No the name `blog_post` doesn't conform to what we recommend and what we use in the docs. You can read more about the conventions used [here](https://www.prisma.io/docs/about/about-the-docs#naming-conventions-for-tables-and-columns).

So in this case the recommended model name would be as I had specified above: `BlogPost`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
